Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Instance/Instance.h

```diff
@@ -184,6 +185,8 @@ namespace AzToolsFramework
         private:
             static constexpr const char s_aliasPathSeparator = '/';
 
+            explicit Instance(Instance* parent, InstanceAlias alias);
```

Comment:
Nitpick: `explicit` is not needed on this constructors as it can't be constructed with a single value.

Reply:
I wanted to suggest the same thing. But I read that using 'explicit' on multi-arg constructor prevents copy-initializer lists: https://stackoverflow.com/questions/39122101/explicit-constructor-taking-multiple-arguments/39122237

I don't think that's the intended user here. So explicit can be removed here I guess.

Reply:
Still use explicit for public constructors based on the link @srikappa-amzn provided.

Reply:
If I understood that article correctly it's actually another reason not to have `explicit` on constructors with multiple arguments. With `explicit` accidental casts are avoided. Now `MyClass x = {"blah", 42}` could be considered a cast of sorts as well, but since it's closer to the syntax `MyClass x{"blah", 42}` I'd personally see it as constructing an object in both cases, hence no `explicit`.